### PR TITLE
[OC-57] Move oc-client-node from OC

### PR DESCRIPTION
**Pull request by nickbalestra**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+matrix:
+  include:
+    - node_js: "4"
+    - node_js: "5"
+    - node_js: "6"
+    - node_js: "7"
+    - node_js: "8"
+
+notifications:
+  email: false
```
Comment (matteofigus) [inline .travis.yml]: 😄

File: README.md [hunk 2]
```diff
@@ -1,2 +1,203 @@
...
+oc-client
+=========
+
+Node.js client for [OpenComponents](https://github.com/opentable/oc)
+
+[![NPM](https://nodei.co/npm/oc-client.png?downloads=true)](https://npmjs.org/package/oc-client)
+
+Node.js version: **4** required
+
+Build status: Linux: [![Build Status](https://secure.travis-ci.org/opentable/oc.png?branch=master)](http://travis-ci.org/opentable/oc) | Windows: [![Build status](https://ci.appveyor.com/api/projects/status/8cklgw4hymutqrsg?svg=true)](https://ci.appveyor.com/project/matteofigus/oc)
```
Comment (matteofigus) [inline README.md]: Update badges links

File: package.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/opencomponents/oc-client-node"
+  },
+  "author": "Matteo Figus",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/opencomponents/oc-client-node/issues"
+  },
+  "homepage": "https://github.com//opencomponents/oc-client-node",
```
Comment (matteofigus) [inline package.json]: //

File: test/fixtures/components/container-with-multiple-nested/_package/server.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+'use strict';
```
Comment (matteofigus) [inline test/fixtures/components/container-with-multiple-nested/_package/server.js]: Can we exclude prettier from _package folders?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "oc-client",
+  "version": "2.1.18",
```
Comment (matteofigus) [inline package.json]: We have 2.1.19 in the other repo. Should this be 2.1.20?

File: test/fixtures/components/container-with-multiple-nested/_package/server.js (hunk 4 above)
Comment (nickbalestra) [inline test/fixtures/components/container-with-multiple-nested/_package/server.js]: Not atm (there is a PR and good progress though toward that -> https://github.com/okonet/lint-staged/pull/188). For the time being I could just make sure we test against `test/acceptance` && `test/unit`

**Pull request merged**